### PR TITLE
nodeController should send events to api server and print to the logging

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>

 /kind bug
/sig node

**What this PR does / why we need it**:
After node is notReady, nodeController will evict pods and delete all pods on that node ,but nodeController does not print  pod deletion events to the logging  and send events to api server.
after this pr, nodeController sends events to api server, and using " kubectl get events"look like 

[root@node-29:/paasdata/op-log/k8s]$ kubectl get events --all-namespaces |grep pod-22-pod-0-1-wh4ml
k8s-test    3m          3m           1         test-2-pod-0-1-wf9mv.155eaee00f864418   Pod                                              Normal    NodeControllerEviction    node-controller           Marking for deleion Pod test-2-pod-0-1-wf9mv from Node 10.200.138.226

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by idealhack** [comment]
it looks like the code hasn't been formatted

**Comment by warmchang** [comment]
@mysunshine92 Good catch, for the code implementation, it looks ok. But there are no alignment due to tab and space, please format and recommit the code.

>

**Comment by mysunshine92** [comment]
@warmchang @idealhack thank you,  I have formatted the code and  have recommitted it.

**Comment by mysunshine92** [comment]
@k82cn PTAL

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go [hunk 1]
```diff
@@ -266,6 +267,13 @@ func NewNodeLifecycleController(
...
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "node-controller"})
+	eventBroadcaster.StartLogging(glog.Infof)
+
+	glog.Infof("Sending events to api server.")
+	eventBroadcaster.StartRecordingToSink(
+		&v1core.EventSinkImpl{
+			Interface: v1core.New(kubeClient.CoreV1().RESTClient()).Events(""),
+		})
 
 	if kubeClient != nil && kubeClient.CoreV1().RESTClient().GetRateLimiter() != nil {
```
Comment (k82cn) [inline pkg/controller/nodelifecycle/node_lifecycle_controller.go]: we do not need to check kubeClient again, I'm ok to handle it in a separated PR.

Comment (mysunshine92) [reply pkg/controller/nodelifecycle/node_lifecycle_controller.go]: @k82cn ok, i will create a new pull request

**Pull request merged**